test_tools/log_parser/parser.py

```diff
@@ -5,8 +5,9 @@
 import reinforcement_learning.messages.flatbuff.v2.PayloadType as pt
 import reinforcement_learning.messages.flatbuff.v2.LearningModeType as lm
 import flatbuffers
-import zstd
+import zstandard as zstd
```

Comment:
Line 37, zstd.decompress(payload) needs to be updated to:
```
dctx = zstd.ZstdDecompressor()
return dctx.decompress(payload)
```

Reply:
You have the wrong pip package installed, the above line works with zstandard